releases/nightly-build/compose-files/docker-compose-nexus-redis.yml

```diff
@@ -266,7 +266,7 @@ services:
       <<: *redis5-variables
     command: |
       /bin/sh -c "
-      until [ -r $${REDIS5_PASSWORD_PATHNAME} ]; do sleep 1; done
+      until [ -r $${REDIS5_PASSWORD_PATHNAME} ] && [ -s $${REDIS5_PASSWORD_PATHNAME} ]; do sleep 1; done
       /usr/local/bin/docker-entrypoint.sh --requirepass `cat $${REDIS5_PASSWORD_PATHNAME}` \
```

Comment:
Suggest that we change the last line to 

```
exec /usr/local/bin/docker-entrypoint.sh ...
```

This makes redis respond to ^C better.